packages/core/template_project/app/src/main/java/TwaDelegationService.java

```diff
@@ -0,0 +1,15 @@
+package <%= packageId %>;
```

Comment:
We don't strictly need to template the TwaDelegationService at this point. eg: we could ship this implemenation with the location library and just swap which implementation we reference on the AndroidManifest.xml.

This approach does make it easier for us to mix and match other extensions, but I wanted to check if that's indeed what we have in mind with this approach.